uuid_test.go

```diff
@@ -530,7 +530,7 @@ func TestNullUUID_UnmarshalJSON(t *testing.T) {
 		},
 		{
 			name:    "Null value",
-			args:    args{val: []byte(`"null"`)},
+			args:    args{val: []byte(`null`)},
```

Comment:
```suggestion
			args:    args{val: []byte("null")},
```

Recommend this for consistency with the corresponding MarshalJSON test case.